src/js/components/Drop/DropContainer.js

```diff
@@ -62,6 +63,13 @@ const DropContainer = forwardRef(
     ]);
     const dropRef = useRef();
     useEffect(() => {
+      const updateAlignControlMirror = () => {
```

Comment:
Agreed with leaving Menu as is. But, can we call this function something like "notifyAlign", since DropContainer doesn't know anything about a "control mirror".

Reply:
Agreed! Addressed!